Source/SwiftLintFramework/Rules/UnusedEnumeratedRule.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  UnusedEnumeratedRule.swift
+//  SwiftLint
+//
+//  Created by Marcelo Fabri on 12/17/16.
+//  Copyright © 2016 Realm. All rights reserved.
+//
+
+import Foundation
+import SourceKittenFramework
+
+public struct UnusedEnumeratedRule: ASTRule, ConfigurationProviderRule {
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "unused_enumerated",
+        name: "Unused Enumerated",
+        description: "When the index is not used, .enumerated() can be removed.",
+        nonTriggeringExamples: [
+            "for (idx, foo) in bar.enumerated() { }\n",
```

Comment:
I think it'd be useful to add a `for (idx, _) in bar.enumerated() { }\n` non-triggering example.